docs/uma-tokenholders/uma-holders.md

```diff
@@ -0,0 +1,36 @@
+---
+title: UMA Tokenholder Responsibilities
+sidebar_label: Responsibilities
+---
+
+Owners of UMA tokens have 2 categories of responsibilities:
+
+1. Voting on price requests from financial contracts using the DVM
+2. Governing the UMA ecosystem by voting on parameter changes and approving system upgrades
+
+UMA tokenholders can use the [Voter dApp](https://vote.umaproject.org/) to vote on price requests. A tutorial on how to vote on price requests from the Voting dApp is [here](uma-tokenholders/voter-dApp.md). 
+
+Voters who participate and vote correctly earn an inflationary reward (currently 0.05% of total network token supply), distributed pro-rata by stake.
+
+## Voting on price requests
+
+Owners of UMA tokens will occasionally be asked to vote on price requests from financial contracts in a 2-stage (commit and reveal) voting period. Tokenholders must reveal their votes or it will not be counted! Each stage is open for 24 hours, so each voting period is 48 hours. 
+
+UMA Tokenholders can discuss their votes in the #voting channel of the UMA Discord before voting.
+
+## Governing the UMA ecosystem
+
+UMA tokenholders govern two areas of the UMA ecosystem:
```

Comment:
Do tokenholders really govern financial contracts? This might be somewhat confusing because they really only govern the DVM but by proxy govern registered financial contracts.

Reply:
I agree it is confusing. In terms of what tokenholders govern I like this wording:

1. Voting on price requests from financial contracts using the DVM
2. Governing the UMA ecosystem by voting on parameter changes and approving system upgrades